src/modules/land_detector/VtolLandDetector.h

```diff
@@ -59,21 +59,23 @@ class VtolLandDetector : public MulticopterLandDetector
 	void _update_topics() override;
 	bool _get_landed_state() override;
 	bool _get_maybe_landed_state() override;
+	bool _get_freefall_state() override;
 
 private:
 
-	uORB::Subscription _airspeed_sub{ORB_ID(airspeed)};
+	uORB::Subscription _airspeed_validated_sub{ORB_ID(airspeed_validated)};
 	uORB::Subscription _vehicle_status_sub{ORB_ID(vehicle_status)};
 
-	airspeed_s _airspeed{};
+	airspeed_validated_s _airspeed_validated{};
 	vehicle_status_s _vehicle_status{};
 
 	bool _was_in_air{false}; /**< indicates whether the vehicle was in the air in the previous iteration */
 	float _airspeed_filtered{0.0f}; /**< low pass filtered airspeed */
 
 	DEFINE_PARAMETERS_CUSTOM_PARENT(
 		MulticopterLandDetector,
-		(ParamFloat<px4::params::LNDFW_AIRSPD_MAX>) _param_lndfw_airspd_max
+		(ParamFloat<px4::params::LNDFW_AIRSPD_MAX>) _param_lndfw_airspd_max,
+		(ParamFloat<px4::params::ASPD_STALL>)  	    _param_aspd_stall
```

Comment:
@dagar Problem: For this PR I used ASPD_STALL in the VTOL land detector, so a parameter from the airspeed module. Now as this module is only built if VTOL/FW, the platform-specific build fails here because all the header files from all separate land detectors are included [here](https://github.com/PX4/Firmware/blob/0d2e5f1c5098fb4318f38e97cfa14d8317c5f4dc/src/modules/land_detector/land_detector_main.cpp#L51). Any easy way to fix that? If not, I would prefer to not introduce new land detector specific parameter but instead set a #define here for the stall speed (eg to 10m/s).